recipe/meta.yaml

```diff
@@ -50,7 +50,8 @@ requirements:
     - libabseil
     - libprotobuf
     - openssl
-    - libcurl
+    # TODO: understand why libcurl >=8.17 causes runtime failures
+    - libcurl <8.17
```

Comment:
We can keep this pin until the issue is fixed as mentioned in https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/1111#pullrequestreview-3479864275.